R/utils.R

```diff
@@ -141,3 +141,33 @@ convert_to_logsd <- function(mean, sd) {
 to_simplex <- function(vector) {
   return(vector / sum(vector))
 }
+
+#' Drop the first element of a simplex and re-normalize the result to sum to 1.
+#'
+#' When this vector corresponds to the generation interval distribution, we
+#' want to drop this first bin. The renewal equation assumes that same-day
+#' infection and onward transmission does not occur, and we assume
+#' everything is 1 indexed not 0 indeced. We need to
+#' manually drop the first element from the PMF vector.
+#'
+#' @param x A numeric vector, sums to 1. Corresponds to a discretized PDF or PMF
+#'   (usually the GI distribution).
+#'
+#' @return A numeric vector, sums to 1.
+#' @export
+#' @examples
+#' pmf_orig <- c(0.1, 0.1, 0.1, 0.7)
+#' pmf_trunc <- drop_first_and_renormalize(pmf_orig)
+drop_first_and_renormalize <- function(x) {
```

Comment:
I see this was copied over to `covid_pmfs.R` but also retained here. Should do one but not both. Unless it's needed elsewhere in the package, I'd keep it in `covid_pmfs.R` for now.

Reply:
Will remove from here